dist/tools/travis-scripts/build_and_test.sh

```diff
@@ -28,6 +28,21 @@ then
     then
         RESULT=0
 
+        if git diff master HEAD -- .travis.yml &> /dev/null; then
+            # check if .travis.yml was changed in the current PR and skip if so
```

Comment:
Huh? I hope changes to the .travis.yml are always intended (otherwise the reviewer does a really bad job). I don't understand your question... This test is testing if the .travis.yml was changed compared to master. If the change happen in the PR that triggered the build (i.e. it is most likely a PR that changed the .travis.yml) this test passes. Otherwise, .travis.yml was changed in master, leading to the problems described in #4920